src/main/java/uk/gov/hmcts/ccd/domain/model/search/CaseSearchResult.java

```diff
@@ -1,25 +1,38 @@
 package uk.gov.hmcts.ccd.domain.model.search;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+
+import java.util.ArrayList;
 import java.util.List;
 
 import static java.util.Collections.emptyList;
 import static java.util.stream.Collectors.toList;
 
-import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
-
 public class CaseSearchResult {
-
     public static final CaseSearchResult EMPTY = new CaseSearchResult(0L, emptyList());
 
     private Long total;
     private List<CaseDetails> cases;
+    @JsonProperty("case_types_results")
```

Comment:
```suggestion
    @JsonProperty("case_type_results")
```
Or change variable name to `caseTypesResults` (whichever is more appropriate for the meaning of this!)

Reply:
DONE

Reply:
we changed the variable name to `caseTypesResults`